### ISSUE TITLE
npm warnings on running npm install

### ISSUE THREAD

**Issue by xyz92** [SharePoint/sp-dev-docs#2893]
#### Category
- [x] Question
- [ ] Typo
- [ ] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior

No npm warnings on running npm install

#### Observed Behavior

npm WARN deprecated @types/adal@1.0.27: '@types/adal' is now '@types/adal-angular'
npm WARN deprecated babel-preset-es2015@6.24.1: 🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read babeljs.io/env to update!
npm WARN deprecated crypto@0.0.3: This package is no longer supported. It's now a built-in Node module. If you've depended on crypto, you should switch to the one that's built-in.
npm WARN deprecated cryptiles@2.0.5: This version is no longer maintained. Please upgrade to the latest version.
npm WARN deprecated scmp@0.0.3: scmp v2 uses improved core crypto comparison since Node v6.6.0
npm WARN deprecated topo@1.1.0: This version is no longer maintained. Please upgrade to the latest version.
npm WARN deprecated boom@2.10.1: This version is no longer maintained. Please upgrade to the latest version.
npm WARN deprecated hoek@2.16.3: This version is no longer maintained. Please upgrade to the latest version.
npm WARN deprecated joi@6.10.1: This version is no longer maintained. Please upgrade to the latest version.
npm WARN deprecated formatio@1.1.1: This package is unmaintained. Use @sinonjs/formatio instead
npm WARN deprecated nomnom@1.6.2: Package no longer supported. Contact support@npmjs.com for more info.
npm WARN deprecated text-encoding@0.6.4: no longer maintained

+-- tslint-react@3.2.0  invalid
`-- UNMET PEER DEPENDENCY tsutils@1.6.0

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules\chokidar\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current:
{"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.3 (node_modules\sane\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current:
{"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.2 (node_modules\watchpack\node_modules\chokidar\node_modules\fsevents):npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current:
{"os":"win32","arch":"x64"})
npm WARN ajv-keywords@3.2.0 requires a peer of ajv@^6.0.0 but none was installed.
npm WARN karma-sinon-chai@1.2.4 requires a peer of sinon@^1.17.2 but none was installed.
npm WARN sinon-chai@2.8.0 requires a peer of sinon@>=1.4.0 <2 but none was installed.
npm WARN tslint-microsoft-contrib@5.0.0 requires a peer of tsutils@1.6.0 but none was installed.
npm WARN ts-jest@22.4.6 requires a peer of jest@^22.4.0 || ^22.5.0-alpha.1 || ^23.0.0-alpha.1 but none was installed.

After upgrading spfx to 1.6.0 and running npm install, I'm seeing the above warnings. How can I fix those warnings?

**Comment by Seiyasu** [comment]
Does it affect the normal operation of the project? Many warning information can be ignored.

**Comment by Seiyasu** [comment]
Close for long time no response. Feel free to contact us to reopen it if needed.

**Comment by rahulEth** [comment]
i am still facing such issue and not able to install other packege..
```npm WARN deprecated bignumber.js@6.0.0: Custom ALPHABET bug fixed in v7.0.2
npm WARN deprecated crypto@1.0.1: This package is no longer supported. It's now a built-in Node module. If you've depended on crypto, you should switch to the one that's built-in.
npm WARN deprecated mongodb@2.2.16: Please upgrade to 2.2.19 or higher
npm WARN deprecated boom@2.10.1: This version is no longer maintained. Please upgrade to the latest version.
npm WARN deprecated cryptiles@2.0.5: This version is no longer maintained. Please upgrade to the latest version.
npm WARN deprecated hoek@2.16.3: This version is no longer maintained. Please upgrade to the latest version.
```
